app/views/devise/sessions/new_with_providers.html.erb

```diff
@@ -0,0 +1,35 @@
+<% if @local_account %>
+  <%= render "devise/shared/form" %>
+<% end %>
+<div class="grid gap-2">
+  <% if @local_account %>
+    <%= render "devise/shared/links" %>
+    <%= button_to t(".return_button"),
+    new_user_session_path,
+    method: :get,
+    params: {
+      local: false
+    },
+    class: "button button--state-default button--size-default w-full" %>
+  <% else %>
+    <%- resource_class.omniauth_providers.each do |provider| %>
+      <%= button_to omniauth_authorize_path(resource_name, provider),
+                    data: {
+                      turbo: false
+                    },
+                    class: "button button--state-default button--size-default w-full" do %>
+        <div class="inline-flex items-center text-center">
+          <%= viral_icon(name: provider, classes: "w-4 h-4 mr-2 -ml-1") %>
+          <%= t(".omniauth", provider: OmniAuth::Utils.camelize(provider)) %>
+        </div>
+      <% end %>
+    <% end %>
+    <%= button_to t(".local_button"),
+    new_user_session_path,
+    method: :get,
+    params: {
```

Comment:
Same comment as above about not needing to use the params: {}

Reply:
Disregard and just leave the params in as this would require a route change

Reply:
As discussed, please see 2817030b69fb2ca9cedc353957f627a8c9f57a51.